processing/src/main/java/io/druid/segment/incremental/IncrementalIndex.java

```diff
@@ -114,8 +114,9 @@ public Long apply(final Object o)
         return null;
       }
       if (o instanceof String) {
+        String s = (String) o;
         try {
-          return Long.valueOf((String) o);
+          return s.isEmpty() ? null : Long.valueOf(s);
```

Comment:
does this change any current behavior that might map empty strings to 0 for numeric values?